docroot/themes/contrib/civictheme/includes/menu.inc

```diff
@@ -14,13 +14,33 @@ function _civictheme_preprocess_block__civictheme_footer_menu(&$variables) {
   }
 
   $menu_name = strtr($variables['elements']['#derivative_plugin_id'], '-', '_');
-  if ($menu_name == 'civictheme-footer') {
+  if ($menu_name == 'civictheme_footer') {
```

Comment:
the machine name is `civictheme-footer` - could you please revert this or provide an explanation why this needs to be changed. Thanks

Reply:
@AlexSkrypnyk  Agree that menu machine name have `-` but  this line of code replacing '-' with '_'
`$menu_name = strtr($variables['elements']['#derivative_plugin_id'], '-', '_');` 

https://github.com/salsadigitalauorg/civictheme_source/pull/468/files#diff-aeb71a4c3adbb084c4326a60ef3d421c2fadf6b687488206120d296dd6cc02e9L16